docs/proposals/README.md

```diff
@@ -22,13 +22,13 @@ Before starting your work and creating a GEP, please take some time to familiari
 general [Gardener Contribution Guidelines](https://gardener.cloud/docs/contribute/).
 
 It is recommended to discuss and outline the motivation of your prospective GEP as a draft with the community before you take the investment of creating the actual GEP. This early briefing supports the understanding for the broad community and leads to a fast feedback for your proposal from the respective experts in the community.
-An appropriate format for this may be the regular [Gardener community meeting](https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md#weekly-meeting).
+An appropriate format for this may be the regular [Gardener community meetings](https://github.com/gardener/documentation/blob/master/website/documentation/contribute/_index.md#bi-weekly-meetings).
```

Comment:
```suggestion
An appropriate format for this may be the regular [Gardener community meetings](https://gardener.cloud/docs/contribute/#bi-weekly-meetings).
```